### PR TITLE
chores: add helper functions

### PR DESCRIPTION
**Pull request by beer-1**
# Description

add helper function interfaces on table and collection modules

---

## Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title, you can find examples of the prefixes below:
  
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] included the necessary unit and integration tests
- [ ] updated the relevant documentation or specification, including comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] confirmed all CI checks have passed

## Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic, API design and naming, documentation is accurate, tests and test coverage

## Summary by CodeRabbit

- New Features
  - Tables now support retrieving all keys and all values, making full-table inspection easier.

- Improvements
  - Governance proposal queries now parse and return normalized fields more consistently, improving clarity and reliability of results.

- Bug Fixes
  - Improved stability of large ordered map insertions, addressing edge cases that could affect data ordering during updates.

- Tests
  - Added a unit test covering proposal retrieval to ensure consistent behavior.

### CODE REVIEW

File: precompile/modules/minitia_stdlib/sources/token/collection.move [hunk 1]
```diff
@@ -471,6 +471,25 @@ module minitia_std::collection {
...
+        event::emit(
+            MutationEvent {
+                collection: mutator_ref.self,
+                mutated_field_name: string::utf8(b"name"),
+                old_value: collection.name,
+                new_value: name
+            }
+        );
+        collection.name = name;
+    }
```
[the remark this answers is not in the document]
Comment (beer-1) [reply precompile/modules/minitia_stdlib/sources/token/collection.move]: removed set_name addition, please resolve it

File: precompile/modules/initia_stdlib/sources/token/collection.move [hunk 2]
```diff
@@ -471,6 +471,25 @@ module initia_std::collection {
...
+        event::emit(
+            MutationEvent {
+                collection: mutator_ref.self,
+                mutated_field_name: string::utf8(b"name"),
+                old_value: collection.name,
+                new_value: name
+            }
+        );
+        collection.name = name;
+    }
```
[the remark this answers is not in the document]
Comment (beer-1) [reply precompile/modules/initia_stdlib/sources/token/collection.move]: removed set_name addition, please resolve it
